Documentation/asagi.rst

```diff
@@ -49,9 +55,42 @@ example on SuperMuc
+   MPI_C_COMPILER                   /lrz/sys/parallel/mpi.ibm/pecurrent/intel/bin/mpicc
+   MPI_C_INCLUDE_PATH               /opt/ibmhpc/pecurrent/mpich2/intel/include64;/opt/ibmhpc/pecurrent/base/include
+   MPI_C_LIBRARIES                  /opt/ibmhpc/pecurrent/mpich2/intel/lib64/libmpi.so;/usr/lib64/libdl.so;/lrz/sys/intel/studio2017_u6/compilers_and_libraries_2017.6.256/linux/compiler/lib/intel64_lin/libirc.so;/usr/lib64/libpthread.so;/usr/lib64/librt.so
+   MPI_C_LINK_FLAGS                  -Wl,--allow-shlib-undefined  -Wl,--enable-new-dtags  -Wl,-rpath,/opt/ibmhpc/pecurrent/mpich2/intel/lib64  -Wl,-rpath,/lrz/sys/intel/studio2017_u6/compilers_and_libraries_2017.6.256/linux/compiler/lib/intel64_lin 
+   MPI_EXTRA_LIBRARY                /opt/ibmhpc/pecurrent/mpich2/intel/lib64/libmpigc4.so;/usr/lib64/libdl.so;/lrz/sys/intel/studio2017_u6/compilers_and_libraries_2017.6.256/linux/compiler/lib/intel64_lin/libirc.so;/usr/lib64/libpthread.so;/usr/lib64/librt.so
+   MPI_Fortran_COMPILER             /lrz/sys/parallel/mpi.ibm/pecurrent/intel/bin/mpif90                                                                                                                             
+   MPI_Fortran_COMPILE_FLAGS                                                                                                                                                                                         
+   MPI_Fortran_INCLUDE_PATH         /opt/ibmhpc/pecurrent/mpich2/intel/include64;/opt/ibmhpc/pecurrent/base/include64 
+   MPI_Fortran_LIBRARIES            /opt/ibmhpc/pecurrent/mpich2/intel/lib64/libmpi.so;/opt/ibmhpc/pecurrent/mpich2/intel/lib64/libmpigf.so;/usr/lib64/libdl.so;/lrz/sys/intel/studio2017_u6/compilers_and_libraries_2017.6.256/linux/compiler/lib/intel64_lin/libirc.so;/usr/lib64/libpthread.so;/usr/lib64/librt.so
+   MPI_Fortran_LINK_FLAGS            -Wl,--allow-shlib-undefined  -Wl,--enable-new-dtags  -Wl,-rpath,/opt/ibmhpc/pecurrent/mpich2/intel/lib64  -Wl,-rpath,/lrz/sys/intel/studio2017_u6/compilers_and_libraries_2017.6.256/linux/compiler/lib/intel64_lin
+   MPI_LIBRARY                      /opt/ibmhpc/pecurrent/mpich2/intel/lib64/libmpi.so                                                                                                                               
+   NETCDF_INCLUDES_C                /lrz/sys/libraries/netcdf/4.3.3/intel/ibmmpi_poe1.4_1505/include                                                                                                                 
+   NETCDF_LIBRARIES_C               /lrz/sys/libraries/netcdf/4.3.3/intel/ibmmpi_poe1.4_1505/lib/libnetcdf.so    
+   PKG_CONFIG_EXECUTABLE            <PATH to ASAGI>/build/lib/pkgconfig/pkg-config   
+ 
+Press ``c`` to configure and ``g`` to generate and exit
+ 
+ .. code-block:: bash
+
```

Comment:
This is very untidy.
@uphoffc Do you have better settings for asagi?

Reply:
I found cleaner settings, see new commits...